src/libs/actions/PersonalDetails.ts

```diff
@@ -157,6 +158,21 @@ function updateDateOfBirth({dob}: DateOfBirthForm) {
     Navigation.goBack();
 }
 
+function updatePhoneNumber(phoneNumber: string) {
+    const parameters: UpdatePhoneNumberParams = {phoneNumber};
+    API.write(WRITE_COMMANDS.UPDATE_PHONE_NUMBER, parameters, {
+        optimisticData: [
```

Comment:
Should we follow the offline pattern B?

cc @mountiny

Reply:
@allgandalf Bump

Reply:
Lets do the same we do for the other private personal details

Reply:
How is the rbr handled there though? Like if there is an error when saving it we still have to show a rbr

Reply:
so if there is an error on providing idk too long legal name, there is no feedback?